AUTOTEST/machine-tux-spack.sh

```diff
@@ -58,6 +58,14 @@ spack load    $spackspec
 spackdir=`spack location -i $spackspec`
 test.sh basic.sh ../src -co: -mo: -spack $spackdir -ro: -superlu
 ./renametest.sh basic $output_dir/basic-dsuperlu
+
+# Clean-up spack build 
+spack spec --yaml $spackspec > test.yaml
+grep ' hash:' test.yaml | sed -e 's/^.*: //' -e 's/^/\//' | xargs spack mark -e
```

Comment:
Why not shorten the `sed` part to the following:

    sed -e 's/^.*: /\//'

Reply:
Yes, that's more concise. Thanks @rfalgout!